definitions/variable/technology/electricity-operation.yaml

```diff
@@ -12,360 +12,82 @@
 #   - Flows in transmission lines
 # Demand not served
 
-- Active Power|Electricity|Biomass:
-    description: Active power of Biomass power plants
-    unit: MWh
-- Active Power|Electricity|Biomass|Traditional:
-    description: Active power of  Traditional Biomass power plants biomass from local
-      and pre-existing resources mostly waste :domestic and agricultural waste, forest
-      residues.
-    unit: MWh
-- Active Power|Electricity|Biomass|New and imported:
-    description: Active power of New+Imported biomass power plants biomass from dedicated
-      energy crops and biomass imports.
-    unit: MWh
-- Active Power|Electricity|Nuclear:
-    description: Active power of nuclear power plants
-    unit: MWh
-- Active Power|Electricity|Gas:
-    description: Active power of Gas power plants
-    unit: MWh
-- Active Power|Electricity|Gas|OCGT:
-    description: Active power of Open Cycle Gas Turbine power plants
-    unit: MWh
-- Active Power|Electricity|Gas|CCGT:
-    description: Active power of Combined Cycle Gas Turbine power plants
-    unit: MWh
-- Active Power|Electricity|Coal:
-    description: Active power of coal power plants
-    unit: MWh
-- Active Power|Electricity|Lignite:
-    description: Active power of Lignite power plants
-    unit: MWh
-- Active Power|Electricity|Oil:
-    description: Active power of oil power plants
-    unit: MWh
-- Active Power|Electricity|Geothermal:
-    description: Active power of Geothermal power plants
-    unit: MWh
-- Active Power|Electricity|Hydro|Reservoir:
-    description: Active power of "simple Hydro reservoir " units
-    unit: MWh
-- Active Power|Electricity|Hydro|Pumped Storage:
-    description: Active power of "Pumped Storage " units
-    unit: MWh
-- Active Power|Electricity|Hydro|Run of River:
-    description: Active power of "Run of River" units
-    unit: MWh
-- Active Power|Electricity|Hydro|Ocean:
-    description: Active power of Ocean power plants
-    unit: MWh
-- Active Power|Electricity|Solar:
-    description: Active power of Photovoltaic power plants
-    unit: MWh
-- Active Power|Electricity|Wind|OnShore:
-    description: Active power of  Wind Onshore power plants
-    unit: MWh
-- Active Power|Electricity|Wind|OffShore:
-    description: Active power of  Wind OffShore power plants
-    unit: MWh
+- Active Power|Electricity|<Fuel>:
+    description: Active power of <this fuel> power plants
+    unit:
+    - MWh
+    - GWh
+
 - Active Power|Electricity|Load Curtailment:
     description: Energy curtailed - from the system point of view is equivalent to
       generation of Active Power
-    unit: MWh
+    unit:
+    - MWh
```

Comment:
Is there a particular reason for allowing multiple units here? Having several units allowed in the database makes working with the data more complicated for users, because they have to keep in mind that orders of magnitude values may differ between different rows.

Reply:
In case study 3, we are dealing with multiple units. In some way, it is suitable to talk about MWh or GWh instead. I made the changes for coherence.
But I understand your point, and I also agree. 
I'll revert this change.